tests/testGaussianPsfFactory.py

```diff
@@ -50,14 +50,14 @@ def testApply(self):
                             naiveSize += 1
                     else:
                         naiveSize = fixedSize
-                    for minSize in (None, max(1, naiveSize-1), naiveSize, naiveSize + 2):
+                    for minSize in (max(1, naiveSize-1), naiveSize, naiveSize + 2):
```

Comment:
It looks like the None iterations (i.e. either of minsize or maxsize are None) don't do anything. Why are they there in the first place?